odoo/addons/base/security/base_groups.xml

```diff
@@ -73,5 +73,19 @@
             <field name="groups_id" eval="[(4, ref('base.group_public'))]"/>
         </record>
 
+        <!-- Default template user for new users signing in -->
+        <record id="template_portal_user_id" model="res.users">
+            <field name="name">Portal User Template</field>
+            <field name="login">portaltemplate</field>
+            <field name="active" eval="False"/>
+            <!-- Avoid auto-including this user in any default group, just like a typical portal member -->
```

Comment:
I know it is a copy-and-paste but the comment does not seem accurate anymore. Let us simply remove it :) .